hack/components.sh

```diff
@@ -79,6 +80,7 @@ rm -rf data/kubemacpool/*
     cd $KUBEMACPOOL_PATH
     kustomize build config/cnao | \
         sed 's/name: kubemacpool-system/name: {{ .Namespace }}/' | \
+        sed 's/namespace: kubemacpool-system/namespace: {{ .Namespace }}/' | \
```

Comment:
Where is this needed?

Reply:
mutatingwebhookconfiguration is not caught by the kustmize so it stays `namespace: kubemacpool-system` unless I add this.

I tried manipulating it in kmp project but this mutatingwebhookconfiguration object does not get attached with the kustomize. investigating